### PR TITLE
Bluetooth: controller: Merge 2 lines

### PR DESCRIPTION
**Pull request by dagbja**
Merge 2 lines that fit on one line

Signed-off-by: Dag Bjarvin <Dag.Bjarvin@nordicsemi.no>

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]

**Review by thoh-ot — APPROVED** [review]

File: subsys/bluetooth/controller/ll_sw/ull_conn.c [hunk 1]
```diff
@@ -4583,8 +4583,7 @@ static inline int length_req_rsp_recv(struct ll_conn *conn, memq_link_t *link,
 			conn->llcp_length.tx_time = eff_tx_time;
 #endif /* CONFIG_BT_CTLR_PHY */
 
-			conn->llcp_length.ack = (conn->llcp_length.req -
-						 1);
+			conn->llcp_length.ack = (conn->llcp_length.req - 1);
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/ull_conn.c]: nitpick change request, remove parenthesis. 

To experiment with, use of git commit --amend and git push -f.

**Review by cvinayak — CHANGES_REQUESTED** [review]

File: subsys/bluetooth/controller/ll_sw/ull_conn.c (hunk 1 above)
Comment (dagbja) [reply subsys/bluetooth/controller/ll_sw/ull_conn.c]: Fixed by amend

**Comment by dagbja** [comment]
Closing request, so I can reopen and re-run timed out tests

**Comment by dagbja** [comment]
Reopening request to re-run timed out tests

**Review by kruithofa — APPROVED** [review]
LGTM

**Review by cvinayak — APPROVED** [review]

**Review by asbjornsabo — APPROVED** [review]
LGTM

**Pull request merged**